### ISSUE TITLE
Insert into Label missing (cget not working)

### ISSUE THREAD

**Issue by GermanYoshi** [TomSchimansky/CustomTkinter#521]
I cant seem to find a way to insert sth. into a label.
Like the .insert from tkinter where you can add text to your existing one.

```
log = self.label_info_1.cget("text")
self.label_info_1.configure(text= log + f'some new text. \n')
```

i tryed it with the mentioned cget but this throws a error
![grafik](https://user-images.githubusercontent.com/39739909/196133632-f80d51e2-7f39-414a-94ac-14ff5f6f3fb8.png)

**Comment by Akascape** [comment]
@GermanYoshi The `cget` method is not implemented. It will be available soon.
Try using the normal method : https://github.com/TomSchimansky/CustomTkinter/issues/27

Or this may work:
`log = self.label_info_1.text`

**Comment by TomSchimansky** [comment]
cget will be implemented in the next version 5.0.0.
